ai/core/tests/core/databricks/test_databricks_integration_tests.py

```diff
@@ -692,3 +693,37 @@ def _helper(x: float) -> int:
         assert "return {" in callable_def and '"result": [' in callable_def
         assert "Args:" in callable_def
         assert "Returns:" in callable_def
+
+
+@requires_databricks
+def test_execute_function_with_custom_client(
+    serverless_client: DatabricksFunctionClient,
+    serverless_client_with_config: DatabricksFunctionClient,
+):
+    with generate_func_name_and_cleanup(serverless_client, schema=SCHEMA) as func_name:
+        function_sample = function_with_string_input(func_name)
+        serverless_client.create_function(sql_function_body=function_sample.sql_body)
+
+        # Client which created the function should execute successfully
+        for input_example in function_sample.inputs:
+            result = serverless_client.execute_function(func_name, input_example)
+            assert result.value == function_sample.output
+
+        # Client created from config should execute successfully
+        for input_example in function_sample.inputs:
+            result = serverless_client_with_config.execute_function(func_name, input_example)
+            assert result.value == function_sample.output
+
+        # Client with fake config should fail as expected
+        function_info = serverless_client.get_function()
```

Comment:
We'll still need to provide a function name here as this is a positional required argument.